Dshop/apps/products_catalogue/tests.py

```diff
@@ -36,11 +36,8 @@ def products():
 
 @pytest.mark.django_db
 def test_generate_xml_file_for_ceneo(products):
```

Comment:
```suggestion
def test_generate_xml_file_for_ceneo(client, products):
```
Do not use django.test.client directly, use pytest-django provided one: https://pytest-django.readthedocs.io/en/latest/helpers.html#client-django-test-client.